### PR TITLE
Additional Knowledge Checks

### PR DESCRIPTION
**Pull request by fortypercenttitanium**
Added some more Knowledge Checks

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The PR title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 
#### 1.Describe the changes made and include why they are necessary or important:

Added some knowledge checks

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

n/a

### CODE REVIEW

**Comment by twalton83** [comment]
Thanks as usual, but can we specify the lesson in cases like this? Thanks!

**Comment by twalton83** [comment]
Also, is this concept specifically covered in the lesson (I’m on mobile)? When these get changed over to the new knowledge checks we need to link to that section.

**Comment by fortypercenttitanium** [comment]
> Also, is this concept specifically covered in the lesson (I’m on mobile)? When these get changed over to the new knowledge checks we need to link to that section.

Which concept specifically? The first knowledge check is about the Box Model, which is covered here. The other two are asking questions similar in scope to two that were already present in the KC.

**Comment by twalton83** [comment]
For clarity,

>  Would you use `margin` or `padding` to increase the size of a button without changing the text size?
> Would you use `margin` or `padding` to adjust how close to the edge of the window an element is?

Are these specifically mentioned in the lesson? The nature of the new knowledge checks system means we need to point to where the answer can be found. As long as one of the resources or concepts covers this, it's fine.

**Comment by fortypercenttitanium** [comment]
> For clarity,
> 
> > Would you use `margin` or `padding` to increase the size of a button without changing the text size?
> > Would you use `margin` or `padding` to adjust how close to the edge of the window an element is?
> 
> Are these specifically mentioned in the lesson? The nature of the new knowledge checks system means we need to point to where the answer can be found. As long as one of the resources or concepts covers this, it's fine.

This is a tough question for me to answer.

By reading the MDN page on the box model, a learner should be able to answer these questions confidently. Are they specifically stated that way on that page? No. But does the page teach you everything you need to know in order to answer this specific question? Absolutely.

I made these questions require a tiny bit of critical thinking. Most questions requiring critical thinking will not have an answer explicitly stated in the material.

If that isn't the direction you want to go with the knowledge checks, I can try to rewrite them to not require extending what was learned about so that we can point to a specific paragraph in the article that answers the question.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Once you change the links to apply the proper knowledge check styling, and have the ones currently just linking to the assignment section link directly to the resource with the answer*, I think this should be good to go.

*and resolve the merge conflict

File: foundations/html_css/the-box-model/the-box-model.md [hunk 1]
```diff
@@ -31,18 +33,23 @@ The only real complication here is that there are multiple ways to manipulate th
...
 2. Because the box model concept is so incredibly fundamental, check out [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). It covers the same material as the video above, but it goes a little further in depth. Pay close attention to the examples and take the time to experiment with their in-browser editor!
 3. [This CSS Tricks page](https://css-tricks.com/almanac/properties/m/margin/) has some further information about the `margin` property that you'll find useful. Specifically, the sections about `auto` and margin collapsing contain things you'll want to know.
 </div>
 
 ### Additional Resources
+
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 ### Knowledge Check
-This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
```
Comment (wise-king-sullyman) [inline foundations/html_css/the-box-model/the-box-model.md:44]: Looks like this needs to be added back in, *then* it should be good to go.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]

File: foundations/html_css/the-box-model/the-box-model.md (hunk 1 above)
Comment (fortypercenttitanium) [reply foundations/html_css/the-box-model/the-box-model.md:44]: Ah, my bad. For some reason the merge conflict was showing something different.

File: foundations/html_css/the-box-model/the-box-model.md [hunk 2]
```diff
@@ -31,18 +33,22 @@ The only real complication here is that there are multiple ways to manipulate th
...
 <div class="lesson-content__panel" markdown="1">
 1. [This video](https://www.youtube.com/watch?v=rIO5326FgPE) is a straightforward overview of the box model, padding and margin. Go ahead and watch this now; it informs everything else.
 2. Because the box model concept is so incredibly fundamental, check out [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). It covers the same material as the video above, but it goes a little further in depth. Pay close attention to the examples and take the time to experiment with their in-browser editor!
 3. [This CSS Tricks page](https://css-tricks.com/almanac/properties/m/margin/) has some further information about the `margin` property that you'll find useful. Specifically, the sections about `auto` and margin collapsing contain things you'll want to know.
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
```
Comment (wise-king-sullyman) [inline foundations/html_css/the-box-model/the-box-model.md:45]: We need this under the Knowledge Check section, not the Additional Resources section, and the Additional Resources preamble placed back in.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]

File: foundations/html_css/the-box-model/the-box-model.md (hunk 2 above)
Comment (fortypercenttitanium) [reply foundations/html_css/the-box-model/the-box-model.md:45]: Ahh, super weird. Sorry about that. For some reason the conflict was showing something different.

Comment (wise-king-sullyman) [reply foundations/html_css/the-box-model/the-box-model.md]: I figured something funky happened haha.

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**